.github/workflows/docker.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le
```

Comment:
https://github.com/dexidp/dex/blob/e74acdff6c68c00029360709c553d301a7850b5f/.github/workflows/artifacts.yaml#L73

The new architecture should also be added to the artifacts build job.

Reply:
updated